go/vt/dbconfigs/dbconfigs.go

```diff
@@ -135,69 +135,102 @@ func registerPerUserFlags(dbc *userConfig, userKey string) {
 
 }
 
+// ConnParams contains mysql.connparams for now
+type ConnParams struct {
```

Comment:
The functionality of this type has changed enough that it's more readable to call this `Connector`.